torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -440,17 +440,21 @@ graph(%a_quant, %normalized_shape, %weight, %bias, %eps, %cudnn_enabled, %output
          return (%r) )";
 
   // ============= General Ops that doesn't require observation =============
-  // aten::avg_pool1d
-  std::string avg_pool1d = R"(
-graph(%a_quant, %kernel_size, %stride, %padding, %ceil_mode, %count_include_pad, %divisor_override):
-          %a_dequant = aten::dequantize(%a_quant)
-          %r = aten::avg_pool1d(%a_dequant, %kernel_size, %stride, %padding, %ceil_mode, %count_include_pad, %divisor_override)
+  // IR pattern common to all general value ops
+  std::string common_general_value_op = R"(
```

Comment:
Maybe call it common_suffix to be more specific?

Reply:
OK

Reply:
will have a larger refactor later that factors out a function, will rename this at that time.